controllers/imageclusterinstall_controller.go

```diff
@@ -246,6 +262,62 @@ func (r *ImageClusterInstallReconciler) Reconcile(ctx context.Context, req ctrl.
 	return ctrl.Result{}, nil
 }
 
+func (r *ImageClusterInstallReconciler) validateSeedReconfigurationWithBMH(
+	ctx context.Context,
+	ici *v1alpha1.ImageClusterInstall,
+	bmh *bmh_v1alpha1.BareMetalHost) (ctrl.Result, error) {
+
+	// no need to validate if inspect annotation is disabled
+	if bmh.ObjectMeta.Annotations != nil && bmh.ObjectMeta.Annotations[inspectAnnotation] == "disabled" {
+		r.Log.Infof("inspection is disabled for BareMetalHost %s/%s, skip hardware validation", bmh.Namespace, bmh.Name)
```

Comment:
This should set the validation condition to `success`/`skipped` with this message